test/integ.awscli-asset.ts.snapshot/manifest.json

```diff
@@ -201,60 +201,6 @@
             "type": "aws:cdk:logicalId",
             "data": "CheckBootstrapVersion"
           }
-        ],
```

Comment:
as per the investigation, these changes were introduced in this pr https://github.com/cdklabs/awscdk-asset-awscli/pull/858, which looks save to be approved, as in that PR, we removed the Python37 Lambda runtime support.